doc/api/Renter.md

```diff
@@ -0,0 +1,280 @@
+| [/renter](#renter-get)                                        | GET       |
+| [/renter](#renter-post)                                       | POST      |
+| [/renter/contracts](#rentercontracts-get)                     | GET       |
+| [/renter/downloads](#renterdownloads-get)                     | GET       |
+| [/renter/files](#renterfiles-get)                             | GET       |
+| [/renter/delete/___*siapath___](#renterdeletesiapath-post)    | POST      |
+| [/renter/download/___*siapath___](#renterdownloadsiapath-get) | GET       |
+| [/renter/rename/___*siapath___](#renterrenamesiapath-post)    | POST      |
+| [/renter/upload/___*siapath___](#renteruploadsiapath-post)    | POST      |
+
+#### /renter [GET]
+
+returns the current settings along with metrics on the renter's spending.
+
+###### JSON Response
+```javascript
+{
+  // Settings that control the behavior of the renter.
+  "settings": {
+    // Allowance dictates how much the renter is allowed to spend in a given
+    // period. Note that funds are spent on both storage and bandwidth.
+    "allowance": {  
+      // Amount of money allocated for contracts. Funds are spent on both
+      // storage and bandwidth.
+      "funds": "1234", // hastings
+
+      // Number of hosts that contracts will be formed with.
+      "hosts":24,
+
+      // Duration of contracts formed, in number of blocks.
+      "period": 6048, // blocks
+
+      // If the current blockheight + the renew window >= the height the
+      // contract is scheduled to end, the contract is renewed automatically.
+      // Is always nonzero.
+      "renewwindow": 3024 // blocks
+    }
+  },
+
+  // Metrics about how much the Renter has spent on storage, uploads, and
+  // downloads.
+  "financialmetrics": {
+    // How much money, in hastings, the Renter has paid into file contracts
+    // formed with hosts. Note that some of this money may be returned to the
+    // Renter when the contract ends. To calculate how much will be returned,
+    // subtract the storage, upload, and download metrics from
+    // ContractSpending.
+    "contractspending": "1234", // hastings
+
+    // Amount of money spent on downloads.
+    "downloadspending": "5678", // hastings
+
+    // Amount of money spend on storage.
+    "storagespending": "1234", // hastings
+
+    // Amount of money spent on uploads.
+    "uploadspending": "5678" // hastings
+  }
+}
+```
+
+#### /renter [POST]
+
+modify settings that control the renter's behavior.
+
+###### Query String Parameters
+```
+// Number of hastings allocated for file contracts in the given period.
+funds // hastings
+
+// Duration of contracts formed. Must be nonzero.
+period // block height
+```
+
+###### Response
+standard success or error response. See
+[API.md#standard-responses](/doc/API.md#standard-responses).
+
+#### /renter/contracts [GET]
+
+// TODO: is this description correct?
+returns all contracts that have been formed by the renter.
```

Comment:
I think it returns only the active contracts. If a contract expires, it will be deleted from the renter and no longer appears.